hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1116,6 +1116,16 @@
           - message
           - session_id
           - created_at
+  select_permissions:
```

Comment:
This was required to give the API role access to the `insert_reconciliation_requests_one` mutation.

> _`insert_<object>_one` will only be available if you have select permissions on the table, as it returns the inserted row_

Docs:  https://hasura.io/docs/latest/mutations/postgres/insert/#insert-a-single-object